.openpublishing.redirection.json

```diff
@@ -2979,6 +2975,10 @@
             "source_path": "docs/standard/serialization/persistent-storage.md",
             "redirect_url": "/dotnet/standard/serialization/serialization-concepts"
         },
+        {
+            "source_path": "docs/standard/serialization/serialization-samples.md",
+            "redirect_url": "/dotnet/standard/serialization/index"
```

Comment:
also wouldn't it be better to redirect to one of the new samples articles and get the traffic and BI data to flow from the old one to one of the new ones?

Reply:
That's an option, but the URL being redirected went to a doc that was intended to list all serialization samples including binary and XML (and now JSON). Rather than go to a specific list of one of those kinds that might not be the group someone was looking for I thought it would be better to go to an overview of all the serialization groupings, and from there you can go down to the Samples TOC node that you're looking for.

Reply:
let's leave as-is then